Basic/Core/Types.pm.PL

```diff
@@ -143,6 +143,15 @@ my @types = (
 		  packtype => 'd*',
 	          defaultbadval => '-DBL_MAX',
 	      },
+	      {
+		  identifier => 'A',
+		  pdlctype => 'PDL_Anyval',
+		  realctype => 'double',
+		  ppforcetype => 'anyval',
+		  usenan => 0, # chm set this to 1, but then get compile error
```

Comment:
This comment needs to be more descriptive or at least the underlying issue should be looked at.